binaries/upload.py

```diff
@@ -94,15 +73,33 @@ def upload_conda_packages():
         required=False,
         help="Specify whether to upload to test PyPI",
     )
+
+    parser.add_argument(
+        "--dry_run",
+        action="store_true",
+        help="dry_run will print the commands that will be run without running them. Only works for pypi now",
+    )
     args = parser.parse_args()
+    print(args)
```

Comment:
should we remove this? Someone might pass a token without realizing there is a print

Reply:
```suggestion
```